.travis.yml

```diff
@@ -11,6 +11,13 @@ cache:
     - $HOME/.cabal/packages
     - $HOME/.cabal/store
 
+
+# We whitelist branches, as we don't really need to build dev-branches.
+# NB: remember to add release branches.
+branches:
+  only:
+    - master
```

Comment:
Yes, and without this change Travis started two builds when you pushed to a branch and then immediately created a PR.